Automattic/wp-calypso
# Created a temporary cart for instantly showing domains at the right panel

Related to 4213-gh-Automattic/dotcom-forge

## Proposed Changes

* Improved multi-domain speed: Videos in this Slack thread: p1700087710110989/1699995887.885749-slack-CKZHG0QCR
* `is-available` is now async and only starts after adding an item in the cart

## Testing Instructions



* Ensure it works at the new multi-domain screens like [/start/domains ](http://calypso.localhost:3000/start/domains)
* Ensure others domains usages like [/domains/add](https://calypso.localhost:3000/domains/add) also works as expected
* Sandbox your API and modify your is-available check to return error following these instructions: p1699996093657959/1699995887.885749-slack-CKZHG0QCR `by adding return 'available_but_not_registrable'; at the beginning of the function.`

## Pre-merge Checklist
